turbo.json

```diff
@@ -88,12 +89,12 @@
             "outputs": []
         },
         "test:unit:browser": {
-            "dependsOn": ["compile:js"],
+            "dependsOn": ["^compile:js"],
```

Comment:
Now, if you run the tests fresh in, say, `@solana/signers` where one of the tests depends on `@solana/addresses`, you'll be sure to have `@solana/addresses` built first so that it can be used in the test.